src/extension.ts

```diff
@@ -37,9 +37,60 @@ export async function activate(
         return;
     }
 
-    const workspacePath = workspaceFolders[0].uri.fsPath;
+    const getCurrentWorkspace = (
+        workspaceFolders: readonly vscode.WorkspaceFolder[]
+    ) => {
+        const activeWorkspace = vscode.window.activeTextEditor
+            ? vscode.workspace.getWorkspaceFolder(
+                  vscode.window.activeTextEditor.document.uri
+              )
+            : undefined;
 
-    const configPaths = configurationService.get<string[]>('configPaths') || [];
+        const workspacePath = activeWorkspace
+            ? activeWorkspace.uri.fsPath
+            : workspaceFolders[0].uri.fsPath;

```

Comment:
Could activeWorkspace above fallback to outputting `workspaceFolders[0]` instead of undefined?

Reply:
yes, ofc :)